### PR TITLE
New sample: NMEA Location Data Source

### PR DESCRIPTION
**Pull request by shubham7109**
This sample demonstrates how to parse NMEA sentences and use the results to show device location on the map. The sample does not implement the non-compulsory option to retrieve real-time NMEA sentences either from a dongle or GPS

Sample Design [link](https://devtopia.esri.com/runtime/common-samples/tree/main/designs/DisplayDeviceLocationWithNMEADataSources).

### CODE REVIEW

File: kotlin/display-device-location-with-nmea-data-sources/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+* LocationDisplay
+* NmeaLocationDataSource
+* NmeaSatelliteInfo
+
+## Offline data
+1. Download the data from [ArcGIS Online](https://arcgis.com/home/item.html?id=d5bad9f4fee9483791e405880fb466da).
+1. Open your command prompt and navigate to the folder where you extracted the contents of the data from step 1.
+1. Execute the following command:
+`adb push Redlands.nmea /Android/data/com.esri.arcgisruntime.sample.displaydevicelocationwithnmeadatasources/files/dodge_city.vtpk`
```
Comment (sorenoid) [inline kotlin/display-device-location-with-nmea-data-sources/README.md:40]:
```suggestion
`adb push Redlands.nmea /sdcard/Android/data/com.esri.arcgisruntime.sample.displaydevicelocationwithnmeadatasources/files/Redlands.nmea`
```

File: kotlin/display-device-location-with-nmea-data-sources/src/main/java/com/esri/arcgisruntime/sample/displaydevicelocationwithnmeadatasources/MainActivity.kt [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+                    line = bufferedReader.readLine()
+                }
+                bufferedReader.close()
+
+                // Set up the accuracy for each location change
+                nmeaLocationDataSource.addLocationChangedListener {
+                    //Convert from Meters to Foot
+                    val horizontalAccuracy = it.location.horizontalAccuracy * 3.28084
+                    val verticalAccuracy = it.location.verticalAccuracy * 3.28084
+                    accuracyTV.text = "Accuracy- Horizontal: %.1fft, Vertical: %.1ft".format(
```
Comment (sorenoid) [inline kotlin/display-device-location-with-nmea-data-sources/src/main/java/com/esri/arcgisruntime/sample/displaydevicelocationwithnmeadatasources/MainActivity.kt:146]:
```suggestion
                    accuracyTV.text = "Accuracy- Horizontal: %.1fft, Vertical: %.1fft".format(
```

**Review by sorenoid — CHANGES_REQUESTED** [review]
Hi @shubham7109 very nice work! I have two small issues, one cosmetic, the other a critical fix to the path to push the nmea file to the device.

**Comment by shubham7109** [comment]
Thanks for the catch Soren! I have gone ahead and added those changes.

**Review by sorenoid — APPROVED** [review]
👍  looks good @shubham7109

**Pull request merged**